deployment-templates/compute-engine/client/forseti-instance-client.py

```diff
@@ -15,6 +15,30 @@
 """Creates a GCE instance template for Forseti Security."""
 
 
+def get_patch_search_expression(forseti_version):
+    """Returns a glob expression matching all patches to the version of passed in parameter
+
+    #TODO update in server/forseti-instance-server if update here
+
+    Args:
+        forseti_version (str): Installed forseti version.  Should start with
+        'tags/v' if patches are to be updated automatically
+
+    Returns:
+        str: Glob expression matching all patches of given forseti_version
+        None: Returns None if forseti_version is not in 'tags/vX.Y.Z' format
+    """
+    if forseti_version[:6] != 'tags/v':
+        return None
+    segments = forseti_version.replace('tags/v', '''''').split('.')
```

Comment:
I don't think that it needs this many `''''''`.  Just 2 would be enough like this `''`.